### PR TITLE
Add RL example

### PR DESCRIPTION
**Pull request by Breakend**
I think this is the rebased version of the previous PR, please double check that this doesn't mess anything up.

### CODE REVIEW

File: docs/src/examples/pytorch_a2c_ppo.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+.. code-block:: bash
+
+    eval.py
+
+.. note ::
+
+  The execution with Oríon does not require the added evaluation methods for
+  a validation set and could use the final training performance. However, for
+  sake if adhering to best practices, we create a validation set method in
```
Comment (bouthilx) [inline docs/src/examples/pytorch_a2c_ppo.rst]: sake if -> sake of

Comment (Breakend) [reply docs/src/examples/pytorch_a2c_ppo.rst]: fixed

File: docs/src/index.rst [hunk 2]
```diff
@@ -13,7 +13,6 @@
    :caption: User's Guide
    :maxdepth: 1
 
-   user/pytorch
```
Comment (bouthilx) [inline docs/src/index.rst]: This is causing an error.

https://travis-ci.org/mila-udem/orion/jobs/418260617#L571

 I would keep this section there even though it is somehow a duplicate.

**Comment by bouthilx** [comment]
Thanks!

**Pull request merged**